ansible/roles/postgresql/templates/pg_hba.conf.j2

```diff
@@ -98,7 +98,7 @@ host    all             all             127.0.0.1/32            md5
 host    all             all             ::1/128                 md5
 # Allow replication connections from localhost, by a user with the
 # replication privilege.
-#local   replication     postgres                                peer
+local   replication     postgres                                peer
```

Comment:
I needed to allow local replication for `pg_basebackup` to be run, which is required as the base for a standby server